packages/plugins/src/plugins/silo/siloRewardFetcher.ts

```diff
@@ -0,0 +1,68 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+import { getEvmClient } from '../../utils/clients';
+import {
+  platformId,
+  siloIncentiveControllerAddresses,
+  rewardVaultAddresses,
+} from './constants';
+import { rewardAbi, rewardTokenAbi } from './abis';
+import { Cache } from '../../Cache';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+
+function fetcher(networkId: EvmNetworkIdType): Fetcher {
+  const executor: FetcherExecutor = async (owner: string, cache: Cache) => {
+    const client = getEvmClient(networkId);
+
+    const rewardCalls = siloIncentiveControllerAddresses.flatMap(
+      (controller) => [
+        {
+          address: controller,
+          abi: rewardAbi,
+          functionName: 'getRewardsBalance',
+          args: [rewardVaultAddresses, owner as `0x${string}`],
```

Comment:
Would this work to avoid the forced cast ?
```suggestion
          args: [rewardVaultAddresses, formatEvmAddress(owner)],
```